sourcegraph/sourcegraph
# Fix browser extension excessive "all websites" permissions for Safari

Closes https://github.com/sourcegraph/sourcegraph/issues/23542.

### How to test

1. Follow [Safari installation](https://github.com/sourcegraph/sourcegraph/blob/ae83ea8a604b8433eb93f7dc8f9f4e6b87137b63/client/browser/README.md#safari)
1. Check that it does not have "all websites" permission
1. Check that per website permission granting works and browser extension injects properly after grant
 

### Screenshots
| Before | After |
| --: | :-- |
| ![image](https://user-images.githubusercontent.com/6717049/140407765-40afe237-1f8e-46f2-bcac-d5305a4a2091.png)| ![image](https://user-images.githubusercontent.com/6717049/140408418-9e18247c-0476-4b3a-9390-1fd96969c73c.png) |
| ![image](https://user-images.githubusercontent.com/6717049/140407661-bd5f9619-a491-419c-8820-59af6d2effd4.png) | ![image](https://user-images.githubusercontent.com/6717049/140406812-bd2f16c2-7e7f-498c-99f5-2dafa0d17d81.png) |
